### DISCUSSION TITLE
point-tracker but with little ArUco markers instead of lights

### DISCUSSION THREAD

**Discussion by risc7** [opentrack/opentrack#1454] [Ideas]
I have been having issues with the ArUco tracker. When the plane of the code is perpendicular to the camera, it glitches out. This makes looking at cockpit instruments and side consoles difficult or impossible. I know point-tracker is far more reliable, but I do not have an infrared camera, so I thought that if I replaced the LEDs on a clip or hat with small ArUco markers or other easily identifiable points could work. The ArUco markers would be treated like the LEDs with the same code for determining head position/rotation as the IR led models.

**Comment by sthalik** [comment]
Try using the color key mode in the PT tracker. You can use brightly-colored pieces of paper as a low-tech tracking clip.
